src/ModFiles/scripts/vscripts/multiplayermod/vars&funcs.nut

```diff
@@ -2099,4 +2103,38 @@ function FindPlayerByName(name) {
         }
     }
     return best
-}
\ No newline at end of file
+}
+
+function StartCountTransition(player) {
```

Comment:
Some statement structures could be optimized here to make it easier to read and run.
For example, instead of surrounding everything, the first if statement can check if the player handle doesn't exist then return. This all can be in one line also.